ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/connected_endpoints.yml

```
@@ -64,6 +66,29 @@ servers:
         port_channel:
           mode: "active"
           description: "PORT_CHANNEL_DESCRIPTION"
+  - name: OLD_SW-1/8
+    adapters:
+      - switches: [ connected_endpoints ]
+        switch_ports: [ Ethernet19 ]
+        endpoint_ports: [ Endpoint_port4 ]
+        mode: trunk
+  - name: OLD_SW-1/9
+    adapters:
+      - switches: [ connected_endpoints, connected_endpoints ]
+        switch_ports: [ Ethernet20, Ethernet21 ]
+        port_channel:
+          mode: active
+          subinterfaces:
+            - number: 100
+            - number: 200
+              vlan_id: 200
+  - name: OLD_SW-1/11
+    adapters:
+      - switches: [connected_endpoints]
+        switch_ports: [Ethernet24]
+        endpoint_ports: [Endpoint_port5]
```

To make coverage 100%
```suggestion
        endpoint_ports: [Endpoint_port5]
        port_channel:
          mode: active
          subinterfaces:
            - short_esi: auto
```

---

This endpoint was added to cover vlans 1-4094. Added a new endpoint with suggested configuration.